How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
```

Comment:
Why Wednesday? Sounds like we are wanting to standardize the release days, etc, which I think is a great idea.